### PR TITLE
[dev] PBI AB#71024: Added MUI Card and Paper Components

### PR DESCRIPTION
**Pull request by cbrewer1689**

### CODE REVIEW

File: src/surfaces/card/card.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+function Card(props: PropTypes) {
+    const {
+        raised = false,
+        ...otherProps
+    } = props;
+
+    return (
+        <MUICard
+            elevation={raised && 22}
```
Comment (groberts314) [inline src/surfaces/card/card.tsx:16]: NAB: I'd prefer a ternary to keep the underlying type correct.
Maybe `elevation={raised ? 22 : undefined}` or whatever an appropriate default for `elevation` is.
Boolean expressions as syntactic sugar for defensive checks is JavaScript silliness IMO; we're using TypeScript and we're beyond that :-P

File: src/surfaces/card/cardBase.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import Card from '@material-ui/core/Card';
+import React from 'react';
+import CardDeprecated from './cardDeprecated';
+
+type PropTypes = {
+    children: React.ReactNode;
+    /**
+     * Dependent on the designVersion number, the component can either use our old Button component
+     * or the new Button component.
```
Comment (groberts314) [inline src/surfaces/card/cardBase.tsx:9]: Button => Card

File: src/surfaces/cardActionArea/cardActionArea.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    const {
+        ...otherProps
+    } = props;
+
+    return (
+        <MUICardActionArea
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...otherProps}
+        />
+    );
```
Comment (groberts314) [inline src/surfaces/cardActionArea/cardActionArea.tsx:14]: Couldn't we skip destructuring `props` into `otherProps` and just just do this?
```JavaScript
return (
    <MUICardActionArea
        // eslint-disable-next-line react/jsx-props-no-spreading
        {...props}
    />
);

File: src/surfaces/cardActions/cardActions.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    const {
+        ...otherProps
+    } = props;
+
+    return (
+        <MUICardActions
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...otherProps}
+        />
+    );
```
Comment (groberts314) [inline src/surfaces/cardActions/cardActions.tsx:14]: Same here; I think we can skip destructuring `props` into `otherProps` and just spread `props` directly in the return JSX.

File: src/surfaces/cardContent/cardContent.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    const {
+        ...otherProps
+    } = props;
+
+    return (
+        <MUICardContent
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...otherProps}
+        />
+    );
```
Comment (groberts314) [inline src/surfaces/cardContent/cardContent.tsx:14]: Same here; I think we can skip destructuring `props` into `otherProps` and just spread `props` directly in the return JSX.

File: src/surfaces/cardHeader/cardHeader.tsx [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    const {
+        ...otherProps
+    } = props;
+
+    return (
+        <MUICardHeader
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...otherProps}
+        />
+    );
```
Comment (groberts314) [inline src/surfaces/cardHeader/cardHeader.tsx:14]: Same here; I think we can skip destructuring `props` into `otherProps` and just spread `props` directly in the return JSX.

File: src/surfaces/cardMedia/cardMedia.tsx [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    const {
+        ...otherProps
+    } = props;
+
+    return (
+        <MUICardMedia
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...otherProps}
+        />
+    );
```
Comment (groberts314) [inline src/surfaces/cardMedia/cardMedia.tsx:14]: Same here; I think we can skip destructuring `props` into `otherProps` and just spread `props` directly in the return JSX.

File: src/surfaces/paper/paper.tsx [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+    const {
+        ...otherProps
+    } = props;
+
+    return (
+        <MUIPaper
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...otherProps}
+        />
+    );
```
Comment (groberts314) [inline src/surfaces/paper/paper.tsx:14]: Same here; I think we can skip destructuring `props` into `otherProps` and just spread `props` directly in the return JSX.

**Review by groberts314 — CHANGES_REQUESTED** [review]
Looks good.  Most change requests are small and cosmetic.

File: src/surfaces/card/card.tsx (hunk 1 above)
Comment (cbrewer1689) [reply src/surfaces/card/card.tsx]: Fixed.

File: src/surfaces/card/cardBase.tsx (hunk 2 above)
Comment (cbrewer1689) [reply src/surfaces/card/cardBase.tsx]: Fixed.

File: src/surfaces/cardActionArea/cardActionArea.tsx (hunk 3 above)
Comment (cbrewer1689) [reply src/surfaces/cardActionArea/cardActionArea.tsx]: Fixed.

File: src/surfaces/cardActions/cardActions.tsx (hunk 4 above)
Comment (cbrewer1689) [reply src/surfaces/cardActions/cardActions.tsx]: Fixed.

File: src/surfaces/cardContent/cardContent.tsx (hunk 5 above)
Comment (cbrewer1689) [reply src/surfaces/cardContent/cardContent.tsx]: Fixed.

File: src/surfaces/cardHeader/cardHeader.tsx (hunk 6 above)
Comment (cbrewer1689) [reply src/surfaces/cardHeader/cardHeader.tsx]: Fixed.

File: src/surfaces/cardMedia/cardMedia.tsx (hunk 7 above)
Comment (cbrewer1689) [reply src/surfaces/cardMedia/cardMedia.tsx:14]: Fixed.

File: src/surfaces/paper/paper.tsx (hunk 8 above)
Comment (cbrewer1689) [reply src/surfaces/paper/paper.tsx]: Fixed.

File: src/surfaces/card/card.tsx (hunk 1 above)
Comment (groberts314) [reply src/surfaces/card/card.tsx]: Nice! Thanks!

File: src/surfaces/card/cardBase.tsx (hunk 2 above)
Comment (groberts314) [reply src/surfaces/card/cardBase.tsx]: Looks good.    Resolving.

File: src/surfaces/cardActionArea/cardActionArea.tsx (hunk 3 above)
Comment (groberts314) [reply src/surfaces/cardActionArea/cardActionArea.tsx]: Looks good.  Resolving.

File: src/surfaces/cardActions/cardActions.tsx (hunk 4 above)
Comment (groberts314) [reply src/surfaces/cardActions/cardActions.tsx]: Looks good.  Resolving.

File: src/surfaces/cardContent/cardContent.tsx (hunk 5 above)
Comment (groberts314) [reply src/surfaces/cardContent/cardContent.tsx]: Looks good.  Resolving.

File: src/surfaces/cardHeader/cardHeader.tsx (hunk 6 above)
Comment (groberts314) [reply src/surfaces/cardHeader/cardHeader.tsx]: Looks good.  Resolving.

File: src/surfaces/cardMedia/cardMedia.tsx (hunk 7 above)
Comment (groberts314) [reply src/surfaces/cardMedia/cardMedia.tsx:14]: Looks good.  Resolving.

File: src/surfaces/paper/paper.tsx (hunk 8 above)
Comment (groberts314) [reply src/surfaces/paper/paper.tsx]: Looks good.  Resolving.

**Review by groberts314 — APPROVED** [review]

**Pull request merged**